validation/frameworks/_cases/WithReturnTypehints.php.expected.json

```diff
@@ -0,0 +1,102 @@
+{
+    "references": {
+        "Fixtures\\Prophecy\\EmptyClass": [
+            "\/Users\/roblou\/code\/php-language-server\/tests\/Validation\/..\/..\/validation\/frameworks\/_cases\/WithReturnTypehints.php"
```

Comment:
I assume this was generated from PHP's `json_encode`, can we use `JSON_UNESCAPED_SLASHES`?
This also should not include an absolute path